anta/tests/aaa.py

```diff
@@ -352,6 +352,7 @@ class VerifyAcctConsoleMethods(AntaTest):
     ```
     """
 
+    description = "Verifies the AAA accounting console method lists for different accounting types (system, exec, commands, dot1x)."
```

Comment:
Why do we need this if it's the same?

Reply:
hmm something with wrap I think - need to investigate again

Reply:
fixed with soft_wrap